src/main/java/com/devcomanda/easymeetup/service/MeetupServiceImpl.java

```diff
@@ -14,6 +14,11 @@ public MeetupServiceImpl(final MeetupRepository meetUpRepository) {
         this.meetUpRepository = meetUpRepository;
     }
 
+    @Override
+    public Meetup findMeetuById(Long id) {
+        return meetUpRepository.getOne(id);
```

Comment:
This method return reference instead of a real meetup entity.
See more here 
https://docs.spring.io/spring-data/jpa/docs/current/api/org/springframework/data/jpa/repository/JpaRepository.html#getOne-ID-
and here
https://www.objectdb.com/api/java/jpa/EntityManager/getReference_Class__Object


You can use findById, that returns optional, if the meetup is not present, optional will be empty and you can use one of the idioms with java optional for resolving that case